firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -13168,7 +13204,6 @@
 				39EF434C260A736D0011E22E /* Experiments */,
 				28EADE5C1AFC3A6D007FB2FB /* Extensions */,
 				C82A94E3269CB74500624AA7 /* FeatureFlags */,
-				F84B21F11A0910F600AAB793 /* Frontend */,
```

Comment:
this has seemed to moved out of alphabetical order, do we still want this change?

Reply:
Ah I see, I didn't know that the ordering of files was reflected in this document! Will revert 🫡